### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by sunshichao0916** [MorrellLAB/BAD_Mutations#54]
Hello BAD_Mutations team,

       When I running the command `./BAD_Mutations.py -v DEBUG compile -P Predictions_Dir -S Test_Data/CBF3.subs 2> Compile.log` , The following errors will appear :
`DEBUG	168 Error:
Traceback (most recent call last):
  File "/vol3/agis/wangli_group/sunshichao/soybean/biosoft/BAD_Mutations/BAD_Mutations.py", line 382, in <module>
    main()
  File "/vol3/agis/wangli_group/sunshichao/soybean/biosoft/BAD_Mutations/BAD_Mutations.py", line 375, in main
    compile_preds(arguments_valid, loglevel)
  File "/vol3/agis/wangli_group/sunshichao/soybean/biosoft/BAD_Mutations/BAD_Mutations.py", line 266, in compile_preds
    parsed_preds = [comp.parse_prediction(rep) for rep in reports]
  File "/vol3/agis/wangli_group/sunshichao/soybean/biosoft/BAD_Mutations/lrt_predict/Predict/hyphy_parser.py", line 133, in parse_prediction
    geneseq += line.strip().split()[8]
IndexError: list index out of range`

 I don't know what went wrong. I hope you can give me some advice.

Thank you so much.

**Comment by TomJKono** [comment]
Hello,

Can you tell me more about the environment you are running in? What version of BAD_Mutations are you running? It looks like you may have an outdated version that is expecting different file formats than the ones that are currently supported. Thank you!

**Comment by sunshichao0916** [comment]
Hello Tom,

Thanks for your help. After I changed the format of the subs file, the above error reporting problem was solved.
But now there is a new problem bothering me, the problem is as follows:

For example: There are six amino acid sites mutated in GLYMA_20G250200 sequence, but the final result only shows the last mutation site (aaPos=91) information. Is there any way to output all the mutation site information?
**Input sub file:**
GLYMA_20G250200.subs
GLYMA_20G250200	635	A	20_47892164
GLYMA_20G250200	482	G	20_47892623
GLYMA_20G250200	447	G	20_47892727
GLYMA_20G250200	300	C	20_47897195
GLYMA_20G250200	278	G	20_47899344
GLYMA_20G250200	91	C	20_47900408

**Output result file:**
GeneID	CDSPos	AlignedPosition	L0	L1	Constraint	Chisquared	P-value	SeqCount	Alignment	ReferenceAA	MaskedConstraint    MaskedP-value	LogisticP_Unmasked	LogisticP_Masked

GLYMA_20G250200	91	1128	-115.084357697778	-105.3928128322451	0.2211826339688233	19.38308973106581	1.069498940964309e-05	60  ILLLLLLLLLLLIIIIIILLLIILLLSLLLLLLLLLLLLRMMMMMMMMMLLLLLLLLLLI	I	0.2214527981159791	1.058092362593577e-05	NA	NA

**Command line:**
`./BAD_Mutations.py -v DEBUG compile -P ./Reports -S ./GLYMA_20G250200.subs 2> GLYMA_20G250200.log`

**log file：**
The last line information:
WARNING	/vol3/agis/wangli_group/sunshichao/soybean/P101SC17040637-01-F004/bad_mutations/compile_test/Reports/Combined_Report.txt exists! Will overwrite!

Looking forward to your reply, thank you.

Best wish,
Shichao
